packages/flutter/lib/src/material/stepper.dart

```diff
@@ -366,6 +367,15 @@ class Stepper extends StatefulWidget {
   /// Overrides the default step icon margin.
   final EdgeInsets? stepIconMargin;
 
+  /// The [Step.content] will be clipped to this Clip type.
+  ///
+  /// Defaults to [Clip.none].
+  ///
+  /// See also:
+  ///
+  ///  * [Clip], which explains how to use this property.
+  final Clip? clipBehavior;
```

Comment:
Why is this nullable?

Reply:
Looking at the existing `clipBehavior` in the framework. some are nullable and some aren't. The main reason for this is a nullable `clipBehavior` can be updated from the widget theme.

<img width="353" alt="Screenshot 2024-07-30 at 19 01 40" src="https://github.com/user-attachments/assets/2428305d-c159-49b9-a1ba-b0beafc95dd7">
<img width="363" alt="Screenshot 2024-07-30 at 19 01 46" src="https://github.com/user-attachments/assets/d8c8ab90-6d20-4392-a2ec-3744ac9fecb9">


Rethinking after your question, maybe this one makes sense to be non-nullable as there is no Stepper theme.

Reply:
Made it non-nullable and updated tests.

Reply:
Thanks for the context! Nullable makes total sense to me when we have a theme - TIL! :)